include/deal.II/grid/reference_cell.h

```diff
@@ -3540,7 +3540,12 @@ ReferenceCell::get_combined_orientation(
           return o;
       }
 
-    Assert(false, (internal::NoPermutation<T>(*this, vertices_0, vertices_1)));
+    // Do not use this in Assert because nvcc when using C++20 assumes that this
```

Comment:
Here and in the other places, can you write this as
```suggestion
    // Do not use 'this' in Assert because nvcc when using C++20 assumes that this
```
to make it clear that "this" isn't just "some thing" but in fact "the 'this' pointer"? I read the comment and thought "what does 'this' refer to here"?